frontend/src/pages/instance/Settings/LauncherSettings.vue

```diff
@@ -8,6 +8,13 @@
                 Flows that perform CPU intensive work may need to increase this from the default of 7500ms.
             </template>
         </FormRow>
+        <FormRow v-if="launcherSupportsAutoSafeMode" v-model="input.disableAutoSafeMode" type="checkbox">
+            Disable Auto Safe Mode
+            <template #description>
+                Prevent Node-RED from automatically entering safe mode when a crash loop is detected.
+                WARNING: Disabling Auto Safe Mode is not recommended. A problem that causes Node-RED to crash multiple successive times may result in a contineous bootloop that will need to be manually resolved.
+            </template>
+        </FormRow>
```

Comment:
This needs a `v-else` block afterwards to show the link to the upgrade instructions

Reply:
Done in incoming update:

![image](https://github.com/user-attachments/assets/b6fba3ee-3bac-4299-9277-bffc5fc94b69)